src/status_im/chat/events/input.cljs

```diff
@@ -413,6 +413,8 @@
  message-model/send-interceptors
  (fn [{{:keys [current-chat-id current-public-key] :as db} :db message-id :random-id current-time :now
        :as cofx} _]
+   (let [wut? nil]
```

Comment:
wut?

Reply:
guys, that's for testing. How otherwise we will know that exception appears in instabug? :)